tests/Doctrine/Tests/Common/Cache/PhpFileCacheTest.php

```diff
@@ -66,6 +66,17 @@ protected function _getCacheDriver()
     {
         return new PhpFileCache($this->directory);
     }
+
+    public function testFailureOnIncludeNotSettedCacheDoesntGetCaughtAsError()
+    {
+        $cache = $this->_getCacheDriver();
+        set_error_handler(function($errno, $errstr, $errfile, $errlin){
+            $this->fail('include failure captured');
```

Comment:
I'll restore the error handler on the capture, it seems to work https://3v4l.org/hn5UC.